files/__main__.py

```diff
@@ -137,12 +137,16 @@ def get_remote_addr():
 	with app.app_context():
 		return request.headers.get('X-Real-IP', default='127.0.0.1')
 
+rate_limiter_enabled = not bool_from_string(environ.get('DBG_LIMITER_DISABLED', False))
+if not rate_limiter_enabled:
+	print("Rate limiter disabled in debug mode!")
```

Comment:
This is good -  might help to also do something similar to remove the 5 second anti-bot check from the signup page when testing

Reply:
Good call, easy enough to do!